conf.py

```diff
@@ -18,19 +18,19 @@
 
 
 # -- Aplus configuration --------------------------------------------------
-course_open_date = '2016-06-01'
-course_close_date = '2020-06-06'
+course_open_date = '2020-04-21'
+course_close_date = '2021-05-01'
 questionnaire_default_submissions = 5
 program_default_submissions = 10
 default_max_group_size = 1
-use_wide_column = True
+use_wide_column = False
 static_host = os.environ.get('STATIC_CONTENT_HOST')
 
 # Define the base URL of the ACOS exercises if the default value is incorrect.
 # The internal IP address of the ACOS container should be used in local testing
 # and in production, the URL of the ACOS production server.
-#acos_submit_base_url = 'https://acos.cs.aalto.fi'
-acos_submit_base_url = 'http://172.21.0.4:3000'
+acos_submit_base_url = 'https://acos.cs.aalto.fi'
```

Comment:
`acos_submit_base_url`: The acos.cs.aalto.fi server shouldn't be used in local testing. The server can not connect back to localhost (teacher's computer and the containers running there). Use the if structure below. It selects between local and production versions.

```
on_container = static_host and '8080' in static_host
if on_container:
    # The static IP address can be defined in docker-compose.yml.
    acos_submit_base_url = 'http://172.21.0.4:3000'
else:
    acos_submit_base_url = 'https://acos.cs.aalto.fi'
```

Reply:
I will include this change in the #13